NEWS.md

```diff
@@ -1,3 +1,24 @@
+# messydates 0.1.2
+
+## Package
+* Added PANARCHIC project details to README file
+* Added cheatsheet
+* Updated documentation
+* Updated website
```

Comment:
Could you be more specific about changes here? For example, "added a new CSS style to website and updated functions displayed"